src/pages/Home/Home.page.jsx

```diff
@@ -1,49 +1,42 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useEffect, useRef } from 'react';
 import Navigation from '../../components/Navigation';
 import ListItems from '../../components/ListItems';
 import ChannelsRow from '../../components/ChannelsRow';
-import useYoutube from '../../utils/hooks/useYoutube';
 import { useMainContext } from '../../state/MainProvider';
-import { useAuth } from '../../providers/Auth';
 import { StyledHomePage, StyledHomePageDivider } from './styled';
+import useYoutube from '../../utils/hooks/useYoutube';
 
 function HomePage() {
-  const { state } = useMainContext();
-  const history = useHistory();
+  const { state, dispatch } = useMainContext();
   const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
   const { searchVideos } = useYoutube();
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+  useEffect(() => {
+    const getVideosFromHook = async () => {
+      const returnedVideos = await searchVideos();
```

Comment:
This searches for `undefined` at first render, as you're not passing your `newSearch` argument to the `searchVideos` function from the hook.